doc/memory.rst

```diff
@@ -37,23 +37,23 @@ such as numpy arrays.
 A simple example:
 ~~~~~~~~~~~~~~~~~
 
-  First we define the cache directory::
+  First define the cache directory::
 
-    >>> cachedir = 'your_cache_dir_goes_here'
+    >>> cachedir = 'your_cache_location_directory'
 
-  We can instantiate a memory context, using this cache directory::
+  A memory context must first be instanciated, using this cache directory::
 
     >>> from joblib import Memory
-    >>> memory = Memory(cachedir=cachedir, verbose=0)
+    >>> memory = Memory(location=cachedir, verbose=0)
 
-  Then we can decorate a function to be cached in this context::
+  Then a function can decorated to be cached in this context::
 
     >>> @memory.cache
     ... def f(x):
     ...     print('Running f(%s)' % x)
     ...     return x
 
-  When we call this function twice with the same argument, it does not
+  When the function is called twice with the same argument, it does not
```

Comment:
Recalling the function with the same argument does not execute it, but loads the output from the pickle file instead.

Reply:
I would say "Calling the function twice with ..."